Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM basex/basexhttp:latest
+COPY ./webapp /srv/basex/webapp
```

Comment:
We need to include a `.basex` file, with local option `CHOP = false`. This will ensure that whitespace isn't chopped from the XML files. 

We can just copy the file from [here](https://github.com/elifesciences/basex-validator-docker/blob/master/src/.basex), because that has [the options](https://github.com/elifesciences/basex-validator-docker/blob/7af2b223243763f80fecf00621c9ae22fe7e17ac/src/.basex#L39-L40) we need. 

[Looks like](https://github.com/elifesciences/basex-validator-docker/blob/7af2b223243763f80fecf00621c9ae22fe7e17ac/Dockerfile#L12) it needs to be copied into `srv/basex` in the docker image.